.github/workflows/ci.yml

```diff
@@ -0,0 +1,42 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+
+  dist:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-python@v2
+      - run: python -m pip install --upgrade pip setuptools wheel twine readme-renderer
+      - uses: actions/checkout@v2
+      - run: python setup.py sdist bdist_wheel
+      - run: python -m twine check dist/*
+
+  pytest:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version:
+          - "3.8"
+          - "3.9"
```

Comment:
Please add more python versions. According to official django supported versions